### ISSUE TITLE
Suppress RuntimeError: (Failure "unknown osd XXX")

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#1019]
## Problem description

When the runtime error occurs whilest purging the asd, it could be that the sync to the DAL is disturbed, leading to inconsistent data
## Possible root of the problem

Albanodecontroller is not catching this exception.
## Possible solution

Catch the exception
## Temporary solution

None
### Setup

Hyperconverged setup
- Two nodes with each six disks for the back-end
### Package information
- openvstorage                          2.7.4-rev.4110.61da5ac-1                                 amd64        openvStorage
- openvstorage-backend                  1.7.4-rev.738.c050b22-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core             1.7.4-rev.738.c050b22-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps          1.7.4-rev.738.c050b22-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin            1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                     2.7.4-rev.4110.61da5ac-1                                 amd64        openvStorage core
- openvstorage-hc                       1.7.4-rev.738.c050b22-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                      1.6.4-rev.380.cfbfc0a-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps                  2.7.4-rev.4110.61da5ac-1                                 amd64        openvStorage Web Applications

**Comment by khenderick** [comment]
Fixed by openvstorage/framework-alba-plugin#238, packaged in openvstorage-backend-1.7.4-rev.748.59bc937

**Comment by JeffreyDevloo** [comment]
### Steps
- Ran the same benchmark which caused the issue to arrise
- Parsed log files to search for the runtime errors
- Found no runtime error instead got:

```
2016-10-11 22:21:11 39100 +0200 - cmp01 - 18093/140064603072320 - lib/albanode - 3397 - WARNING - Could not match ASD EPyoPwBVt1m8MVN4GR3DqqnJKExbvEga to any backend. Cannot purge
```

### Test result

Test passed.

### Packages
- openvstorage                          2.7.4-rev.4115.c0dda6a-1                                 amd64        openvStorage
- openvstorage-backend                  1.7.4-rev.748.59bc937-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core             1.7.4-rev.748.59bc937-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps          1.7.4-rev.748.59bc937-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin            1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                     2.7.4-rev.4115.c0dda6a-1                                 amd64        openvStorage core
- openvstorage-hc                       1.7.4-rev.748.59bc937-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                      1.6.4-rev.383.24dfaaa-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps                  2.7.4-rev.4115.c0dda6a-1                                 amd64        openvStorage Web Applications
